### ISSUE TITLE
[FEAT] Card deck and ability to shuffel

### ISSUE THREAD

**Issue by codesnippy** [Kruptein/PlanarAlly#887]
**Is your feature request related to a problem? Please describe.**
Would like to see a option to build a card deck that can me shuffled and where a card can be taken from the deck.
 

**Describe the solution you'd like**
Make use of the template feature and the ability to set image variants to construct a complete deck. Fist image depicting the deck back's. The rest are the card fronts.
Then something along the lines of dice trowing to select a random card from the deck that makes a copy of the template with the selected card front displayed.

This way I could setup more types of games with Planarally.

**Comment by develroo** [comment]
Must admit, I have been recently introduced into Deadlands and so Cards would be a very nice addition. Optional of course.. so you can toggle it on and off as the Game requires. Does not need fancy animation just to be able to keep track of a full pack of cards and have a random shuffle ability.

**Comment by Kruptein** [comment]
This is a very niche feature, I'll keep the ticket open for a moment that I have some time to do this, but it's not super high priority for me atm.

I'm always open for PRs though ;) But I guess I would need to have some more modular support for addons for that kind of contribution.

**Comment by develroo** [comment]
Yeah.. not sure how it would be implemented, other than it could be some sort of general matrix which can hold plugin values on a temporary or permanent basis. 

As for the card logic and animation, well there is already this [Card deck](https://github.com/deck-of-cards/deck-of-cards) in JavaScript and CSS, and he even has [a nice demo](https://deck.of.cards/) of how it can look like in a multiplayer arrangement.


So maybe you can find a way of crowbarring that in, when you have time ;)
